Resources/docs/introduction.rst

```diff
@@ -103,7 +103,7 @@ Place the following mapping file in the
     <massive-search-mapping xmlns="http://massive.io/schema/dic/massive-search-mapping">
 
         <mapping class="Model\Product">
-            <index name="product" />
+            <index value="product" />
```

Comment:
Because the `FieldEvaluator` evaluates the property differently based on the attribute name. I only adjusted the index tag to the other ones, where name is handled differently than on the index tag.